### DISCUSSION TITLE
the from address does not match a verified sender identity

### DISCUSSION THREAD

**Discussion by endersonrufino** [balta-io/forum#301] [Tira Dúvidas] [answered]
@andrebaltieri eu estou recendo o seguinte erro ao tentar enviar um email usando o sendgrid: the from address does not match a verified sender identity, eu olhei perguntas anteriores e uma delas você dizia que devia usar o mesmo email do cadastro no sendgrid, eu já fiz isso e continua com o mesmo erro, fiz algumas pesquisas e ainda não consegui resolver. Se puder me ajuidar.

obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @endersonrufino, como vai?
Esta thread aqui vai te ajudar:
https://github.com/balta-io/forum/discussions/259

Caso tenha problemas com SSL, aqui tem a solução também:
https://github.com/balta-io/forum/discussions?discussions_q=sendgrid
